### PR TITLE
 Performance problems fixes. Part 1

### PR DESCRIPTION
**Pull request by mandrono**

### CODE REVIEW

**Comment by mandrono** [comment]
@dmitry-gorokhov @nshchego take a look please

File: inference-engine/src/mkldnn_plugin/mkldnn_node.cpp [hunk 1]
```diff
@@ -1281,19 +1283,19 @@ MKLDNNNode* MKLDNNNode::NodesFactory::create(const std::shared_ptr<ngraph::Node>
...
+    //         if (ol != nullptr && ol->created(extMgr))
+    //             newNode = ol.release();
+    //     } catch (const InferenceEngine::Exception& ex) {
+    //         if (ex.getStatus() != NOT_IMPLEMENTED) {
+    //             throw;
+    //         } else {
+    //             errorMessage += getExceptionDescWithoutStatus(ex);
+    //         }
+    //     }
+    // }
```
Comment (nshchego) [inline inference-engine/src/mkldnn_plugin/mkldnn_node.cpp:1298]: Why?

Comment (mandrono) [reply inference-engine/src/mkldnn_plugin/mkldnn_node.cpp:1298]: it's needed to have correct performance report (without performance drop when unsupported node fallback on ngraph implementation)

File: inference-engine/src/mkldnn_plugin/nodes/mkldnn_crop_node.cpp [hunk 2]
```diff
@@ -1,198 +0,0 @@
-// Copyright (C) 2018-2021 Intel Corporation
```
Comment (nshchego) [inline inference-engine/src/mkldnn_plugin/nodes/mkldnn_crop_node.cpp:1]: Why it was not removed by rebasing on master?

Comment (mandrono) [reply inference-engine/src/mkldnn_plugin/nodes/mkldnn_crop_node.cpp:1]: i have no idea
but on master this files is absent

File: inference-engine/src/mkldnn_plugin/mkldnn_node.cpp (hunk 1 above)
Comment (mandrono) [reply inference-engine/src/mkldnn_plugin/mkldnn_node.cpp:1299]: comment added

File: inference-engine/src/mkldnn_plugin/mkldnn_graph_optimizer.cpp [hunk 3]
```diff
@@ -903,6 +906,54 @@ void MKLDNNGraphOptimizer::FuseConvolutionAndDWConvolution(MKLDNNGraph &graph) {
     // }
 }
 
+void MKLDNNGraphOptimizer::FuseConvolutionAndSimpleOperationThroughMaxPool(MKLDNNGraph &graph) {
```
[the remark this answers is not in the document]
Comment (mandrono) [reply inference-engine/src/mkldnn_plugin/mkldnn_graph_optimizer.cpp:910]: as we discussed, will be update дфеук

File: inference-engine/src/mkldnn_plugin/nodes/cum_sum.cpp [hunk 4]
```diff
@@ -22,65 +25,81 @@ class CumSumImpl: public ExtLayerBase {
...
+                    IE_THROW() << "CumSum layer with name '" << layerName << "' doesn't support 'axis' input tensor with non scalar rank";
             }
 
-            if (dataShape != layer->outData[0]->getTensorDesc().getDims())
+            if (dataShape != cumsum->get_output_shape(0))
                 IE_THROW() << "CumSum layer with name '" << layerName << "' has different 'data' input and output dimensions";
 
             shape = dataShape;
 
             LayerConfig config;
```
[the remark this answers is not in the document]
Comment (mandrono) [reply inference-engine/src/mkldnn_plugin/nodes/cum_sum.cpp]: switched

File: inference-engine/src/mkldnn_plugin/nodes/mkldnn_fullyconnected_node.cpp [hunk 5]
```diff
@@ -24,7 +24,9 @@ bool MKLDNNFullyConnectedNode::isSupportedOperation(const std::shared_ptr<ngraph
             errorMessage = "Only legacy FullyConnected operation is supported";
             return false;
         }
-        if (std::dynamic_pointer_cast<const ngraph::opset1::Constant>(fc->get_input_node_shared_ptr(WEIGHTS_ID)) == nullptr) {
+        if (std::dynamic_pointer_cast<const ngraph::opset1::Constant>(fc->get_input_node_shared_ptr(WEIGHTS_ID)) == nullptr &&
```
[the remark this answers is not in the document]
Comment (mandrono) [reply inference-engine/src/mkldnn_plugin/nodes/mkldnn_fullyconnected_node.cpp]: removed

**Pull request merged**